### ISSUE TITLE
[BUG] Click install error on Ubuntu 18.04 LTS

### ISSUE THREAD

**Issue by robwedd** [Chia-Network/chia-blockchain#1452]
**Describe the bug**
`sh install.sh` fails to install `click`. Several reinstall attempts failed at the same point. Manually installing `click` using `pip` fixes the bug.

```
Collecting click@ https://github.com/Chia-Network/click/tarball/master#egg=package-1.0
  Using cached https://github.com/Chia-Network/click/tarball/master (322 kB)
    ERROR: Command errored out with exit status 1:
     command: ~/git/chia-blockchain/venv/bin/python3.7 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-29zsnqd7/click_130493b0e26b49da937b82aeecddad9b/setup.py'"'"'; __file__='"'"'/tmp/pip-install-29zsnqd7/click_130493b0e26b49da937b82aeecddad9b/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-c_1eolzz
         cwd: /tmp/pip-install-29zsnqd7/click_130493b0e26b49da937b82aeecddad9b/
    Complete output (28 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-29zsnqd7/click_130493b0e26b49da937b82aeecddad9b/setup.py", line 3, in <module>
        setup(name="click", install_requires=["colorama; platform_system == 'Windows'"])
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.7/distutils/core.py", line 121, in setup
        dist.parse_config_files()
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/dist.py", line 494, in parse_config_files
        ignore_option_errors=ignore_option_errors)
      File "~git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 106, in parse_configuration
        meta.parse()
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 382, in parse
        section_parser_method(section_options)
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 355, in parse_section
        self[name] = value
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 173, in __setitem__
        value = parser(value)
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 430, in _parse_version
        version = self._parse_attr(value)
      File "~/git/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 305, in _parse_attr
        module = import_module(module_name)
      File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'click'
    ----------------------------------------
WARNING: Discarding https://github.com/Chia-Network/click/tarball/master#egg=package-1.0. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Collecting chiabip158==1.0
  Using cached chiabip158-1.0-cp37-cp37m-manylinux2010_x86_64.whl (2.3 MB)
Collecting aiosqlite==0.17.0
  Using cached aiosqlite-0.17.0-py3-none-any.whl (15 kB)
ERROR: Could not find a version that satisfies the requirement click (unavailable) (from chia-blockchain)
ERROR: No matching distribution found for click (unavailable)
```

The line from the above:
```
      File "/tmp/pip-install-29zsnqd7/click_130493b0e26b49da937b82aeecddad9b/setup.py", line 3, in <module>
        setup(name="click", install_requires=["colorama; platform_system == 'Windows'"])
```
seems to indicate that this is somehow getting confused with a windows installation.

Manually installing:

`pip install click`

fixes the problem.

**Desktop (please complete the following information):**

- OS: Linux
- OS Version/Flavor:Ubuntu 18.04 LTS
- CPU:Intel® Core™ i5-4300U CPU

**Additional context**

**Comment by hging** [comment]
same errors

**Comment by hoffmang9** [comment]
Will be fixed in the 1.0.1 release very shortly

**Comment by erdnapa** [comment]
Manually installing click does NOT fix the issue.:

```
chip@7085c26e4c9c:~/chia-blockchain$ pip install -U click
Collecting click
  Using cached https://files.pythonhosted.org/packages/d2/3d/fa76db83bf75c4f8d338c2fd15c8d33fdd7ad23a9b5e57eb6c5de26b430e/click-7.1.2-py2.py3-none-any.whl
Installing collected packages: click
Successfully installed click-7.1.2
chip@7085c26e4c9c:~/chia-blockchain$ sh install.sh 
apt-get is /usr/bin/apt-get
(standard_in) 1: syntax error
pacman: not found
yum: not found
yum: not found
Python version is 3.7
Requirement already satisfied: pip in ./venv/lib/python3.7/site-packages (21.0.1)
Requirement already satisfied: wheel in ./venv/lib/python3.7/site-packages (0.36.2)
Looking in indexes: https://pypi.org/simple, https://pypi.chia.net/simple/
Requirement already satisfied: miniupnpc==2.1 in ./venv/lib/python3.7/site-packages (2.1)
Looking in indexes: https://pypi.org/simple, https://pypi.chia.net/simple/
Obtaining file:///home/chip/chia-blockchain
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Collecting click@ https://github.com/Chia-Network/click/tarball/master#egg=package-1.0
  Using cached https://github.com/Chia-Network/click/tarball/master (322 kB)
    ERROR: Command errored out with exit status 1:
     command: /home/chip/chia-blockchain/venv/bin/python3.7 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-q2vnt6z7/click_213c1ea5caa5443aade1d47872c434d4/setup.py'"'"'; __file__='"'"'/tmp/pip-install-q2vnt6z7/click_213c1ea5caa5443aade1d47872c434d4/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-jnvep501
         cwd: /tmp/pip-install-q2vnt6z7/click_213c1ea5caa5443aade1d47872c434d4/
    Complete output (28 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-q2vnt6z7/click_213c1ea5caa5443aade1d47872c434d4/setup.py", line 3, in <module>
        setup(name="click", install_requires=["colorama; platform_system == 'Windows'"])
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/__init__.py", line 129, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.7/distutils/core.py", line 121, in setup
        dist.parse_config_files()
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/dist.py", line 494, in parse_config_files
        ignore_option_errors=ignore_option_errors)
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 106, in parse_configuration
        meta.parse()
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 382, in parse
        section_parser_method(section_options)
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 355, in parse_section
        self[name] = value
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 173, in __setitem__
        value = parser(value)
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 430, in _parse_version
        version = self._parse_attr(value)
      File "/home/chip/chia-blockchain/venv/lib/python3.7/site-packages/setuptools/config.py", line 305, in _parse_attr
        module = import_module(module_name)
      File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'click'
    ----------------------------------------
WARNING: Discarding https://github.com/Chia-Network/click/tarball/master#egg=package-1.0. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Collecting blspy==1.0
  Using cached blspy-1.0-cp37-cp37m-manylinux2010_x86_64.whl (820 kB)
Collecting setproctitle==1.2.2
  Using cached setproctitle-1.2.2-cp37-cp37m-manylinux1_x86_64.whl (36 kB)
Collecting clvm-tools==0.4.3
  Using cached clvm_tools-0.4.3-py3-none-any.whl (36 kB)
Collecting sortedcontainers==2.3.0
  Using cached sortedcontainers-2.3.0-py2.py3-none-any.whl (29 kB)
Collecting colorlog==4.7.2
  Using cached colorlog-4.7.2-py2.py3-none-any.whl (10 kB)
ERROR: Could not find a version that satisfies the requirement click (unavailable) (from chia-blockchain)
ERROR: No matching distribution found for click (unavailable)
```

**Comment by mariano54** [comment]
Fixed in 1.0.1
